expression/builtin_string.go

```diff
@@ -1502,33 +1502,26 @@ type charLengthFunctionClass struct {
 }
 
 func (c *charLengthFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
-	sig := &builtinCharLengthSig{newBaseBuiltinFunc(args, ctx)}
+	bf, err := newBaseBuiltinFuncWithTp(args, ctx, tpInt, tpString)
```

Comment:
done. It seems that 'verifyArgs(args)' is not done first in many builtin functions. 'LENGTH' for example. Maybe need refine